.github/workflows/xpu.yml

```diff
@@ -0,0 +1,24 @@
+name: xpu
+
+on:
+  push:
+    tags:
+      - ciflow/xpu/*
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref_name }}-${{ github.ref_type == 'branch' && github.sha }}-${{ github.event_name == 'workflow_dispatch' }}-${{ github.event_name == 'schedule' }}
+  cancel-in-progress: true
+
+jobs:
+  linux-jammy-xpu-py3_8-build:
+    name: linux-jammy-xpu-py3.8
+    uses: ./.github/workflows/_linux-build.yml
+    with:
+      build-environment: linux-jammy-xpu-py3.8
+      docker-image-name: pytorch-linux-jammy-xpu-2024.0-py3
+      runner: linux.2xlarge
```

Comment:
Runner type should probably be ``linux.idc.xpu``

Reply:
Sorry for the misleading in RFC, the xpu build also can use aws instance directly.